tools/f18/CMakeLists.txt

```diff
@@ -13,11 +6,15 @@ add_executable(f18
   f18.cpp
   dump.cpp
 )
+set_property(TARGET f18 PROPERTY CXX_STANDARD 17)
+target_compile_features(f18 PRIVATE cxx_std_17)
+
 
 target_link_libraries(f18
   FortranParser
   FortranEvaluate
   FortranSemantics
+  FortranDecimal
```

Comment:
I see the following in origin/master. Does this patch need a rebase before further review?

target_link_libraries(f18
  FortranParser
  FortranEvaluate
  FortranSemantics
  FortranLower
  LLVMSupport
)

Reply:
Working on a rebase -- testing across multiple platforms, compilers, and optimization levels  has revealed some issues.